client/packages/host/src/Admin/ElectronSettings.tsx

```diff
@@ -84,7 +89,7 @@ export const ElectronSettings = () => {
 
   useEffect(() => {
     electronNativeAPI?.linkedBarcodeScannerDevice().then(setScanner);
-  }, [electronNativeAPI]);
+  }, [electronNativeAPI, scannerType]);
```

Comment:
when scannerType is refreshed we should update scanner (as connected etc..), this is already done in context, but didn't want to change too much